charts/testkube-api/templates/testworkflows/builtin-templates/cypress.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.global.testWorkflows.createOfficialTemplates }}
+kind: TestWorkflowTemplate
+apiVersion: testworkflows.testkube.io/v1
+metadata:
+  name: official--cypress--beta
+  labels:
+    testkube.io/name: Cypress
+    testkube.io/wizard: enabled
+  annotations:
+    testkube.io/categories: E2E
+    testkube.io/description: Run Cypress tests
+    testkube.io/icon: cypress
+spec:
+  config:
+    dependencies_command:
+      description: Command to install dependencies
+      type: string
+      default: npm install
+    version:
+      description: Cypress version to use
+      type: string
+      default: 13.6.4
+    run:
+      description: Run command
+      type: string
+      default: "npx cypress run"
+  steps:
+  - name: Install dependencies
+    container:
+      image: "cypress/included:{{ config.version }}"
+    shell: "{{ config.dependencies_command }}"
+  - name: Run Cypress tests
+    container:
+      image: "cypress/included:{{ config.version }}"
+    shell: "{{ config.run }}"
```

Comment:
Same as above, `run.shell` is available. Same for all below

Reply:
updated (all templates)